tensorflow/lite/micro/static_vector.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TENSORFLOW_LITE_MICRO_STATIC_VECTOR_H_
+#define TENSORFLOW_LITE_MICRO_STATIC_VECTOR_H_
+
+#include <array>
+#include <cassert>
+#include <cstddef>
+#include <ostream>
```

Comment:
Can we avoid ostream usage? I have found just the inclusion of the i/o stream headers results in binary bloat.

Reply:
Sure, that's only there for the operator<<() supplied as a convenience at the bottom. I'll axe it, and either put that in a separate header or just inline something in the one place that used operator<<().

(That lightweight operator<<() is a little perilous anyhow; it doesn't handle all the possible side-effects that using other operator<<()s within its implementation can have on the ostream state.)